src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -14,6 +14,7 @@
 public class FeedbackRubricQuestionDetails extends FeedbackQuestionDetails {
     public int numOfRubricChoices;
     public List<String> rubricChoices;
+    public List<Double> rubricWeights;
```

Comment:
I am not sure how to test it correctly. I have no problems editing and showing the results of the rubric question in "Session with different question types"(one of the default sessions in the dev server).